definitions/variable/emissions/emissions.yaml

```
@@ -113,10 +113,9 @@
       - Emissions|{Level-3 Species}|AFOLU|Land|Harvested Wood Products
       - Emissions|{Level-3 Species}|AFOLU|Land|Other
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions from land that is covered or saturated by water for
-      all or part of the year (e.g., peatland) and that does not fall
-      into the forest land, cropland, grassland or settlements
-      categories (partially IPCC 1996 category 4D/5A/5B/5E; IPCC 2006 category 3B4)
+    description: Net emissions from managed wetlands (e.g., peatland), including 
+      emissions from drained wetlands used for agriculture, forestry or other uses (IPCC 2006 categories 3B1biii, 3B2biii and 3B3biii, 3B5biv, 3B6biv), 
+      and emissions from rewetted wetlands (IPCC 2006 category 3B4).
```

No full stop at the end of descriptions for consistency.
```suggestion
      and emissions from rewetted wetlands (IPCC 2006 category 3B4)
```